superset-frontend/src/SqlLab/components/ExploreCtasResultsButton/index.jsx

```diff
@@ -68,39 +62,35 @@ class ExploreCtasResultsButton extends React.PureComponent {
           all_columns: [],
           row_limit: 1000,
         };
-        this.props.actions.addInfoToast(
-          t('Creating a data source and creating a new tab'),
-        );
+        addInfoToast(t('Creating a data source and creating a new tab'));
 
         // open new window for data visualization
         exploreChart(formData);
       })
       .catch(() => {
-        this.props.actions.addDangerToast(
-          this.props.errorMessage || t('An error occurred'),
-        );
+        addDangerToast(errorMessage || t('An error occurred'));
       });
-  }
+  };
 
-  render() {
-    return (
-      <>
-        <Button
-          buttonSize="small"
-          onClick={this.onClick}
-          tooltip={t('Explore the result set in the data exploration view')}
-        >
-          <InfoTooltipWithTrigger
-            icon="line-chart"
-            placement="top"
-            label="explore"
-          />{' '}
-          {t('Explore')}
-        </Button>
-      </>
-    );
-  }
-}
+  const onClick = () => {
+    visualize();
+  };
+
+  return (
+    <Button
+      buttonSize="small"
+      onClick={onClick}
```

Comment:
```suggestion
  return (
    <Button
      buttonSize="small"
      onClick={visualize}
```
Since this is only running visualize(), you can delete this function and pass visualize() straight into the onClick attribute.